packages/pagination/src/Pagination.tsx

```diff
@@ -31,7 +31,7 @@ export function Pagination({
 					case 'direction':
 						return (
 							<PaginationItemDirection
-								key={idx}
+								key={item.direction}
```

Comment:
Since we know there is only 1 element for each direction, we can use the direction as the key will improve render performance.